Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC1001.AzureDevOpsPersonalAccessToken_pats.sarif

```diff
@@ -79,7 +79,7 @@
           "message": {
             "id": "Default",
             "arguments": [
-              "VariousPats.txt",
+              "SEC1001.AzureDevOpsPersonalAccessToken_pats.txt",
```

Comment:
I added a reference to "VariousPats.txt" at my PR #95 
just a reminder I need to update once your merged